### PR TITLE
VAGOV-5963: Use Packagist for va-gov/web release management and updates. 

### PR DESCRIPTION
**Pull request by jonpugh**
# Goal

Automatically update CMS composer when WEB releases are made.

# How

This PR removes the pseudo-repo from CMS composer.json, and uses a packagist repo instead:

https://packagist.org/packages/va-gov/web

- `composer update` automatic updates with one command. 
- Removes the need to create a special devops job to read WEB releases and make the change with a script or something.
- Removes the need for a symlink to `/web`: Composer installers now puts the package in `/web` for us.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -164,14 +152,15 @@
...
+            "npm-asset",
+            "node-project"
         ],
         "installer-paths": {
             "docroot/core": [
                 "type:drupal-core"
             ],
             "docroot/libraries/{$name}": [
-                "type:drupal-library",
+                "type:drupal-lfbrary",
```
Comment (ElijahLynn) [inline composer.json]: Typo made it in

**Comment by jonpugh** [comment]
It's working! ...with a caveat: 

Run `composer update` and if there is a new SHA, it will be set in composer.lock.

If you run composer update again, it will set composer.lock to the branch instead of the SHA.

I think I can inspect the package more to prevent this.

**Comment by ElijahLynn** [comment]
Closing to destroy and rebuild anew.

**Comment by ElijahLynn** [comment]
I changed the base branch from `develop` to `master` as we switched our CD to be based on the GitHub Flow model (as opposed to Git Flow) and `develop` is now deprecated (https://github.com/department-of-veterans-affairs/va.gov-cms/pull/617). It may be best instead of fixing conflicts to rebase/replay your commits on top of current master:

`git fetch --all`
`git rebase origin/master`
`git push --force`

**Comment by jonpugh** [comment]
Hey check this out...
<details>
    <summary>What if... </summary>

There was a standard way to do drop down  "more info" things?
</details>

**Comment by jonpugh** [comment]
Hey @ethanteague ...

Content check failed. I'm commenting out here.

We can merge fix later, but tests are probably going to fail on all branches right now.

https://www.va.gov/health-care/
#    And I should see "GI Bill School Feedback"

**Comment by jonpugh** [comment]
Hey @ethanteague I wonder if you could give this a review and help figure out the behat failure?

The behat step that rebuilds web is failing:

```
      │ Step 30 start: Parse HTML files
      │ Step 30 end [7193ms]: Parse HTML files
      │ 
      │ Step 31 start: Add nonce to script tags
      │ Step 31 end [573ms]: Add nonce to script tags
      │ 
      │ Step 32 start: Update external links
      │ Step 32 end [174ms]: Update external links
      │ 
      │ Step 33 start: Add IDs to subheadings
      │ Step 33 end [270ms]: Add IDs to subheadings
      │ 
      │ Step 34 start: Check for broken links

      The command cd .. && composer va:web:build returned a non-zero exit code. (Exception)
    │
    │  Step Failed. 
    │  Site: pr589.ci.cms.va.gov 
    │  Current URL: http://pr589.ci.cms.va.gov/sections
    │  Last Page Output: http://pr589.ci.cms.va.gov//sites/pr589.ci.cms.va.gov/files/test_failures/failure-1571274417.html 
```

**Comment by ElijahLynn** [comment]
Corresponds to https://github.com/department-of-veterans-affairs/vets-website/pull/10803

**Comment by jonpugh** [comment]
Ok, it is the --fail-fast flag...

@ElijahLynn I propose we remove the change in test hook so we only update the SHA?

**Comment by jonpugh** [comment]
Follow up PR: Add `--drupal-fail-fast` flag that doesn't fail when there are broken links

**Comment by ElijahLynn** [comment]
We also need to remove the custom repositories key in composer.json @jonpugh, I just remembered.

**Comment by ElijahLynn** [comment]
#651

**Pull request merged**